scripts/postshrinkwrap.js

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+// Postprocesses the lockfile to remove C++ dependency and to enforce https://
+
+const fs = require('fs');
+const path = require('path');
+const lockfile = require('../npm-shrinkwrap');
+
+// Force all installations of Dredd to use only the pure JavaScript version
+// of the API Blueprint parser. It has slower performance, but it solves
+// roughly a billions of installation & distribution problems.
```

Comment:
```suggestion
// roughly billions of installation & distribution problems.
```

Not quite billions either but ok 😄

Reply:
I think the `a` got in because it was `a billion`, but then I changed it to `billions` 😂

Reply:
Oh I forgot to commit this. Will do in a follow-up.